docs/quickstart_guide.md

```diff
@@ -149,7 +179,7 @@ SSD-equipped nodes are highly recommended. Full list of CloudLab nodes can be fo
     ```bash
     sudo screen -dmS containerd bash -c "containerd > >(tee -a /tmp/vhive-logs/containerd.stdout) 2> >(tee -a /tmp/vhive-logs/containerd.stderr >&2)"
     ```
-2. Run the script that creates the multinode cluster:
+2. Run the script that creates the multinode cluster (without `stargz`):
     ```bash
     ./scripts/cluster/create_multinode_cluster.sh > >(tee -a /tmp/vhive-logs/create_multinode_cluster.stdout) 2> >(tee -a /tmp/vhive-logs/create_multinode_cluster.stderr >&2)
```

Comment:
Command does not copy properly by clicking copy botton. 
The command copies like this.
```
./scripts/cluster/create_multinode_cluster.sh stock-only > >(tee -a /tmp/vhive-logs/
create_multinode_cluster.stdout) 2> >(tee -a /tmp/vhive-logs/create_multinode_cluster.stderr >&2)
```

Reply:
Thank you! I have changed it now (some indentation problem). Please let me know if it still persists.